docs/engineering/technical-planning.md

```diff
@@ -0,0 +1,36 @@
+# Technical Planning Docs
+## Request for Comment
+In the course of planning development, there are often scenarios where a technology or implementation decision fork is presented and an engineer wants to solicit ideas or feedback. 
+
+Writing a full-blown Architectural Decision Record may not appropriate as that document requires significant overhead in proposing answers that still may be largely unknown. In most cases, we only want to invest the time in writing an ADR only when a decision has been made and needs to be codified. Until then, it's a Decision Proposal, not a Decision Record.
```

Comment:
>Writing a full-blown Architectural Decision Record may not appropriate as that document requires significant overhead in proposing answers that still may be largely unknown.

found a grammar error (missing "be"), and perhaps split into 2 sentences:

Writing a full-blown Architectural Decision Record may not be appropriate. ADR documents require significant overhead in proposing answers that still may be largely unknown.

Reply:
thanks on the missing `be`, will fix. The conjunction in that sentence is intentional.